Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayAuthorizationDelegate.m

```diff
@@ -123,6 +123,9 @@ -(void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController
 	self.checkout.shippingAddress = [self buyAddressWithABRecord:address];
 	if ([self.checkout.shippingAddress isValidAddressForShippingRates]) {
 		[self updateCheckoutWithAddressCompletion:completion];
+	} else {
```

Comment:
This entire if statement is the same as in lines 210-215, could be extracted into a common method